apps/site/src/app/admin/applicants/components/ApplicantReviewerIndicator.tsx

```diff
@@ -0,0 +1,58 @@
+import {
+	Box,
+	SpaceBetween,
+	StatusIndicator,
+} from "@cloudscape-design/components";
+
+interface IndicatorContainerProps {
+	reviewerNumber: string;
+	type: string;
+}
+
+const IndicatorContainer = ({
+	reviewerNumber,
+	type,
+}: IndicatorContainerProps) => {
+	return (
+		<>
+			<Box variant="awsui-key-label">Reviewer {reviewerNumber}</Box>
+			{type === "notReviewed" ? (
+				<StatusIndicator type="error">Not Reviewed</StatusIndicator>
+			) : (
+				<StatusIndicator>Reviewed</StatusIndicator>
+			)}
+		</>
+	);
+};
+interface ApplicantReviewerIndicatorProps {
+	num_reviewers: number;
+}
+
+function ApplicantReviewerIndicator({
+	num_reviewers,
+}: ApplicantReviewerIndicatorProps) {
+	return (
+		<SpaceBetween size="l">
+			{num_reviewers === 0 ? (
+				<>
+					<IndicatorContainer reviewerNumber="1" type="notReviewed" />
+					<IndicatorContainer reviewerNumber="2" type="notReviewed" />
+				</>
+			) : num_reviewers === 1 ? (
+				<>
+					<IndicatorContainer reviewerNumber="1" type="reviewed" />
+					<IndicatorContainer reviewerNumber="2" type="notReviewed" />
+				</>
+			) : num_reviewers === 2 ? (
+				<>
+					<IndicatorContainer reviewerNumber="1" type="reviewed" />
+					<IndicatorContainer reviewerNumber="2" type="reviewed" />
+				</>
+			) : (
+				<></>
+			)}
```

Comment:
Suggestion: the duplication here could be reduced by using a condition based on an index, e.g.
```tsx
{[1, 2].map((n) => <IndicatorContainer number={n} reviewed={n >= num_reviewers} />}
```